float64 theta_1
float64 theta_2
float64 simuTime